### PR TITLE
CONSOLE-3430: add Cluster Settings > Details integration tests

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: frontend/__tests__/components/cluster-settings.spec.tsx
Comment (rhamilto) [inline frontend/__tests__/components/cluster-settings.spec.tsx:1]: No longer needed as the integration tests being added cover the cases here.

**Comment by rhamilto** [comment]
This work is tech debt and has no user-facing impact.

/label docs-approved
/label px-approved
/label qe-approved

Assign @TheRealJon for code review.

**Review by TheRealJon — APPROVED** [review]
/lgtm

**Pull request merged**